docs/developer_documentation/ref-rest2.md

```diff
@@ -92,6 +92,25 @@ Key | Type | Description
 --- | --- | ---
 *includeCategories* | boolean | Includes a list of categorical options in attribute metadata for CATEGORICAL and CATEGORICAL_MREF attributes
 
+### Sorting categorical options
+You can add a unique, visible attribute to the metadata of the table after the ID-attribute to sort the categorical options.
+
+**Attribute definition**
+
+| id | visible | unique | type |
+|----|---------|--------|------|
+| sortOrder | TRUE | TRUE | int |
```

Comment:
How about renaming the example id to ```mySortAttribute``` to clarify that ```sortOrder``` isn't an id with a special meaning?